<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QComboBox" name="checkFilesComboBox">
     <item>
      <property name="text">
       <string>Images &amp; Videos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Videos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Images</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All files</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" rowspan="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Similarity Threshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Check files</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="similarityThresholdLabel">
     <property name="text">
      <string>50 - Fairly similar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File Settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSlider" name="similarityThresholdHorizontalSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>similarityThresholdHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>updateSimilarityThresholdLabel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Preferences</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>applySettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Preferences</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>cancelSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateSimilarityThresholdLabel(int)</slot>
  <slot>applySettings()</slot>
  <slot>cancelSettings()</slot>
 </slots>
</ui>
